src/constants/landing.js

```diff
@@ -2,6 +2,8 @@ import { FACEBOOK_WIDGET_API, GOOGLE_PLUS_ONE_WIDGET_API, TWITTER_WIDGET_API } f
 
 export const GFW_ASSETS_PATH = process.env.REACT_APP_GFW_ASSETS_PATH;
 
+export const DOWNLOAD_APK_LINK = "https://rink.hockeyapp.net/api/2/apps/2394979ddd9d415c9b0e466656757d5d/app_versions/9?format=apk&avtoken=510e2758876fcd8bbd275f0eff3ff33bb3e36984&mctoken=18aa5ac7b7a92bf4840c3eb4ef096d054bdff8c8"
```

Comment:
Needs to be an Env var so we can update it without changing code.

Reply:
👍